packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec

```diff
@@ -0,0 +1,101 @@
+# Generated from foreman_inventory_upload-0.0.1.dev1.gem by gem2rpm -*- rpm-spec -*-
+# template: foreman_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_inventory_upload
+%global plugin_name inventory_upload
+%global foreman_min_version 1.20
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1.dev1
+Release: 1%{?foremandist}%{?dist}
+Summary: Adds ability to upload hosts managed in Foreman to Red Hat cloud inventory
+Group: Applications/Systems
+License: GPLv3
+URL: https://github.com/theforeman/foreman_inventory_upload
```

Comment:
Not blocking the merge, but when do you plan to transfer the repo?

Reply:
In a near future, I assume in the next couple of days